src/utils/database.utils.js

```diff
@@ -0,0 +1,64 @@
+import { Op } from 'sequelize';
```

Comment:
Am not really we need to create this file here, we can all these in the controller files.

Reply:
Okay. Why I abstracted it is because the controller was about 100 lines long and i felt it was difficult to read and understand. Should i still move it?

Reply:
I have moved the implementation outside back in the controller